packages/synthesizer/__tests__/euclidean.sty

```diff
@@ -0,0 +1,511 @@
+Colors {
```

Comment:
should the tests use the `examples/` versions instead of copying the code here?

Reply:
These files are specific to the examples I'm trying to do, so no. The changes to the Euclidean style are improvements that don't cause regressions, so I'm inclined to leave it in.

Existing examples are okay, the two-vectors example here:
![image](https://user-images.githubusercontent.com/11740102/119286749-b195a000-bc0a-11eb-8ad8-c5579a7b4afd.png)